src/screens/channels/channels.tsx

```diff
@@ -57,19 +57,39 @@ export default class ChannelsScreen extends React.Component<ChannelsScreenProps,
 
   fetchChannels = async() => {
     try {
+
+      if(this.state.refreshing){
+        this.setState({channels: []})
+      }
+      await Permissions.askAsync(Permissions.LOCATION);
+      const location = await Location.getCurrentPositionAsync({ enableHighAccuracy: true });
+      
       const response = await client.query<any>({
+        variables: {
+          take: 10,
+          skip: 0,
```

Comment:
Apesar de tu já tá mandando ignoreIds, o ideal é que tu mande o skip tbm

this.state.channels.length